charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/etcd.yaml

```diff
@@ -8,7 +8,7 @@ https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-promet
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  namespace: {{ template "kube-prometheus-stack.namespace" . }}
+  namespace: {{ template "grafana.namespace" . }}
```

Comment:
These files are generated from a python script - please update this and re-run to regenerate the dashboards.

Reply:
@vsliouniaev Done. Please re-trigger the CI.